src/main/java/com/github/technus/tectech/mechanics/enderStorage/EnderFluidContainer.java

```diff
@@ -10,27 +11,24 @@
 
 public class EnderFluidContainer implements IFluidHandler, Serializable {
     private static final int CAPACITY = 64000;
-    private int fluidID = -1;
-    private int fluidQuantity = 0;
+    private NBTTagCompound fluid;
 
     public EnderFluidContainer() {
     }
 
     private FluidStack getFluidStack() {
         FluidStack fluidStack = null;
-        if (fluidID >= 0) {
-            fluidStack = new FluidStack(fluidID, fluidQuantity);
+        if (fluid != null) {
+            fluidStack =  FluidStack.loadFluidStackFromNBT(fluid);
```

Comment:
it will remove the exised fluid setting in cover of the old saves. you should mark the fluid id as deprecated and transform the id into fluid tag if the id existed